src/Auth0Client.ts

```diff
@@ -378,7 +385,13 @@ export default class Auth0Client {
       this.worker
     );
 
-    const decodedToken = this._verifyIdToken(authResult.id_token, nonceIn);
+    const organizationId = options.organization || this.options.organization;
```

Comment:
As we are expecting `this.options.organization`, shouldn't we add `organizationId` to the `Auth0ClientOptions` ?

Reply:
We will in time - as the feature is not yet available to the public, we should not confuse developers by making this too visible in the docs or API, but that will come in a future PR.